lambdas/reconciliation-report-migration/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/reconciliation-report-migration",
-  "version": "19.0.0",
+  "version": "19.1.0",
```

Comment:
Oh good catch. This package is only in the es-phase-2 branch and wouldn't have been caught by the latest release's version bump.